deepfence_frontend/apps/dashboard/src/features/common/data-component/searchCloudAccountsApiLoader.tsx

```diff
@@ -43,23 +43,35 @@ export const searchCloudAccountsApiLoader = async ({
 
 export const useGetCloudAccountsList = ({
   nodeType,
+  fetchOnAction,
 }: {
   // TODO: strongly type this
   nodeType: string;
+  fetchOnAction?: boolean;
```

Comment:
lets not take a flag here. make nodeType optional, if a node type supplied then make request or do nothing.